src/hypothesis/strategies.py

```diff
@@ -804,9 +804,8 @@ def dm_func(denom):
 
 @cacheable
 @defines_strategy
-def decimals(
-    min_value=None, max_value=None, allow_nan=None, allow_infinity=None
-):
+def decimals(min_value=None, max_value=None, allow_nan=None,
+             allow_infinity=None, places=None, precision=None):
```

Comment:
I think if you go with this interface now, it will be impossible to add bounds later, so this is a permanent decision.